src/app/workflow/refresh-workflow-organization/refresh-workflow-organization.component.ts

```diff
@@ -54,6 +58,28 @@ export class RefreshWorkflowOrganizationComponent extends RefreshOrganizationCom
     this.isRefreshing$ = this.alertQuery.showInfo$;
   }
 
+  ngOnChanges() {
+    this.isGitHubOrg = this.orgWorkflowObject.sourceControl === 'github.com';
+  }
+
+  openConfirmationDialog() {
+    const confirmationDialogData: ConfirmationDialogData = {
+      title: 'Refresh Organization',
+      message: `Are you sure you wish to refresh the organization?
+                It has no effect on workflows already being synchronized via the Dockstore GitHub App.`,
```

Comment:
I forget the term for this, but add what it does do as opposed to what it doesn't do (or do both). 

Something like "Are you sure you wish to refresh the organization? This will sequentially refresh workflows that have been added through the Dockstore UI and will have no effect on workflows already being synchronized via the Dockstore GitHub App" except maybe better

Reply:
I think it's getting a little too wordy. It'll also affect workflows added through CLI too.

Reply:
:thinking:

in the meantime, you should probably take-out/change the tooltip too if you have not already 
![Screenshot from 2021-11-04 17-36-59](https://user-images.githubusercontent.com/1730679/140423792-5e86b531-3a34-4d36-b8ce-35d12c3c092a.png)

Reply:
Removed the tooltip as per slack discussion.

"Are you sure you wish to refresh the organization? This will sequentially refresh workflows that have been added through the Dockstore website/CLI and will have no effect on workflows already being synchronized via the Dockstore GitHub App"

is too wordy imo, doesn't provide any additional useful information and could possibly be missing other sources. Others should probably chime in (and provide reasoning)

Reply:
How about: "This will sequentially refresh all workflows in the organization except for those already being synchronized via the Dockstore GitHub App."?

Reply:
That works for me